src/components/Login/Login.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../_variables.scss';
```

Comment:
Really like the styling on the form.

Reply:
Sweet, maybe it's something we can continue in the rest of the forms, then?